src/lib/middleware/morgan.ts

```diff
@@ -26,12 +26,22 @@ function skipAssets(req: ArtsyRequest, res: ArtsyResponse): boolean {
   )
 }
 
+const BANNED_PARAMS = ["setup_intent_client_secret"]
+export function filterParams(url: string): string {
+  const cleaned = BANNED_PARAMS.reduce((prevUrl, key) => {
+    const re = new RegExp(`${key}=([^&]+)`)
```

Comment:
Also #superminor, reading this reducer code would be a bit easier if `key` -> `bannedParam` and similar to the comment below we don't abbreviate variables:

```tsx
const paramToMask = new RegExp(`${bannedParam}=([^&]+)`)
const maskedUrl = prevUrl.replace(paramToMask, (match, p1) => match.replace(p1, "[FILTERED]"))
return maskedUrl 
```

```tsx

```